src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -1388,7 +1387,7 @@ uint32_t ApplicationManagerImpl::GetNextMobileCorrelationID() {
     mobile_corelation_id_ = 0;
   }
 
-  return mobile_corelation_id_;
+  return mobile_corelation_id_.load();
```

Comment:
@ychernysheva @AKalinich-Luxoft do we need this change there? Wouldn't it work as a regular access operation if we do not specify the memory order?

Reply:
@LitvinenkoIra I think that we can skip this `load()` call

Reply:
@ychernysheva please use either `load()` + `store()` or regular `operator=` and copy ctor.
Both implementations should have the same behavior.

Reply:
@LitvinenkoIra @AKalinich-Luxoft I remove `load()` call for consistency in a30f13f